examples/data_collect/main.cpp

```diff
@@ -790,29 +575,6 @@ int main(int argc, char *argv[]) {
 #else
         char out_file[MAX_FILE_PATH_SIZE];
 
-        if (!fsf_flag) {
-            snprintf(out_file, sizeof(out_file), "%s/%s_frame_%s_%05u.bin",
```

Comment:
I think we need to keep this if we want to save the frames without multi threading.

Reply:
Will replace it without the if condition